winui/Kanban/column.md

```diff
@@ -0,0 +1,172 @@
+
+<kanban:SfKanban x:Name="kanban"
+                 AutoGenerateColumns="False" 
+                 ItemsSource="{Binding TaskDetails}">
+        <kanban:KanbanColumn HeaderText="To Do" Categories="Open" />
+        <kanban:KanbanColumn HeaderText="In Progress" Categories="In Progress" />
+        <kanban:KanbanColumn HeaderText="Code Review" Categories="Code Review" />
+        <kanban:KanbanColumn HeaderText="Done" Categories="Done, Validated" />
+</kanban:SfKanban>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="4" %}
+
+this.kanban.Columns.Add(new KanbanColumn() { HeaderText = "To Do", Categories = "Open" });
+this.kanban.Columns.Add(new KanbanColumn() { HeaderText = "In Progress", Categories = "In Progress" });
+this.kanban.Columns.Add(new KanbanColumn() { HeaderText = "Code Review", Categories = "Code Review" });
+this.kanban.Columns.Add(new KanbanColumn() { HeaderText = "Done", Categories = "Done, Validated" });
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customizing column headers
+
+### Customizing column header text
+
+You can set the header text for a column using the `HeaderText` property, which allows you to display simple text as the column title.
+
+{% tabs %}
+{% highlight XAML %}
+
+<kanban:KanbanColumn Categories="Open" HeaderText="To Do"/>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+this.kanban.Columns.Add(new KanbanColumn() { Categories = "Open", HeaderText = "To Do" });
+
+{% endhighlight %}
+{% endtabs %}
+
+### Customizing column header template
+
+To replace the UI of the header, you can use the `ColumnHeaderTemplate` of the `SfKanban`. The following code example illustrates how this can be done.
+
+{% tabs %}
+{% highlight XAML %}
+
+<kanban:SfKanban x:Name="kanban"
+                 ItemsSource="{Binding TaskDetails}">
+    <kanban:SfKanban.ColumnHeaderTemplate>
+        <DataTemplate>
+            <StackPanel Background="LightBlue">
+                <TextBlock Margin="10" Text="{Binding  HeaderText}" Foreground="Red" HorizontalAlignment="Center"/>
+            </StackPanel>
+        </DataTemplate>
+    </kanban:SfKanban.ColumnHeaderTemplate>
+    <kanban:SfKanban.DataContext>
+        <local:ViewModel/>
+    </kanban:SfKanban.DataContext>
+</kanban:SfKanban>
+
+{% endhighlight %}
+{% endtabs %}
+
+## Expanding or collapsing column
+
+To expand or collapse a column programmatically, use the `KanbanColumn.IsExpanded` property. Set this property to `true` to expand the column or `false` to collapse it.
+
+{% tabs %}
+{% highlight XAML %}
+
+<kanban:KanbanColumn Categories="Open" HeaderText="To Do" IsExpanded="False"/>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+this.kanban.Columns.Add(new KanbanColumn() { Categories = "Open", HeaderText = "To Do", IsExpanded = false });
+
+{% endhighlight %}
+{% endtabs %}
+
+## Enabling or disabling drag-and-drop
+
+The WinUI Kanban Control allows you to enable or disable the drag-and-drop functionality for cards using the `KanbanColumn.AllowDrag` property.
```

Comment:
Cards can be dragged and dropped across columns, within columns, or across swimlane rows to modify the card's status. To enable drag-and-drop functionality within the kanban, set the `AllowDrag` property of `KanbanColumn` to true. To disable drag-and-drop, set this property to false. The default value is true.